src/tasks/runstart.ts

```diff
@@ -927,121 +930,55 @@ export const RunStartQuest: Quest = {
-        get("_monstersMapped") >= 3 ||
-        have($item`cherry`) ||
-        (() => {
-          // if we have another skeleton in the ice house, we don't need to map a novelty skeleton
-          const banishes = get("banishedMonsters").split(":");
-          const iceHouseIndex = banishes.map((string) => string.toLowerCase()).indexOf("ice house");
-          if (iceHouseIndex === -1) return false;
-          return ["remaindered skeleton", "factory-irregular skeleton", "swarm of skulls"].includes(
-            banishes[iceHouseIndex - 1],
-          );
-        })(),
-      do: () => mapMonster($location`The Skeleton Store`, $monster`novelty tropical skeleton`),
-      combat: new CombatStrategy().macro(
-        Macro.if_(
-          "!haseffect Everything Looks Yellow",
-          Macro.if_(
-            $monster`novelty tropical skeleton`,
-            Macro.externalIf(useParkaSpit, Macro.trySkill($skill`Spit jurassic acid`))
-              .trySkill($skill`Blow the Yellow Candle!`)
-              .tryItem($item`yellow rocket`),
-          ),
-        ).abort(),
-      ),
-      outfit: () => ({
-        ...baseOutfit(false),
-        shirt: useParkaSpit ? $item`Jurassic Parka` : undefined,
-        offhand: $item`Roman Candelabra`,
-        modifier: `${baseOutfit().modifier}, -equip miniature crystal ball, -equip Kramco Sausage-o-Matic™`,
-      }),
-      post: (): void => {
-        if (have($item`MayDay™ supply package`) && !get("instant_saveMayday", false))
-          use($item`MayDay™ supply package`, 1);
-        if (have($item`space blanket`)) autosell($item`space blanket`, 1);
-      },
-      limit: { tries: 1 },
-    },
-    {
-      name: "Novelty Tropical Skeleton",
-      prepare: (): void => {
-        if (useParkaSpit) {
-          cliExecute("parka dilophosaur");
-        } else if (!have($item`yellow rocket`) && !have($effect`Everything Looks Yellow`)) {
-          if (myMeat() < 250) throw new Error("Insufficient Meat to purchase yellow rocket!");
-          buy($item`yellow rocket`, 1);
+        // handle Peridot setup
+        if (force?.name === "Peridot of Peril") {
+          PeridotOfPeril.setChoice($monster`novelty tropical skeleton`);
+          unbreakableUmbrella();
+          if (haveEquipped($item`miniature crystal ball`)) equip($slot`familiar`, $item.none);
         }
-        if (have($item`Roman Candelabra`) && !have($effect`Everything Looks Yellow`)) {
-          equip($slot`offhand`, $item`Roman Candelabra`);
-        } else {
+        // handle Map prep
+        if (force?.name === "Map the Monsters") {
           unbreakableUmbrella();
+          if (haveEquipped($item`miniature crystal ball`)) equip($slot`familiar`, $item.none);
+        }
+        // fallback prep
+        if (!force) {
+          if (have($item`Roman Candelabra`) && !have($effect`Everything Looks Yellow`))
+            equip($slot`offhand`, $item`Roman Candelabra`);
+          else unbreakableUmbrella();
```

Comment:
If we do eventually check for ELY prior to equipping the candelabra in the outfit part of the code, then we can do a simple/similar check for equipping the unbreakable umbrella here. This probably also can then be pulled out of the conditional branches